### PR TITLE
forgot password ui

### PR DESCRIPTION
**Pull request by fazlerabby07**

### CODE REVIEW

**Comment by sifatul** [comment]
@fazlerabby07 bhai for image use this  src = 'url(https://source.unsplash.com/random/?developer,office)',
git e atto boro image store korle pore free usage e ses hoye jabe

File: pages/auth/forgot-password.jsx [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+            <FlexItemStyled>
+                <Image alt='' src={imageUrl} />
+            </FlexItemStyled>
+            <FlexItemStyled>
+                <ForgotItemsStyled>
+                    <TitleStyled>Forgot password?</TitleStyled>
+                    <DescriptionStyled>
+                        Insert your email to receive the reset link
+                    </DescriptionStyled>
+                    <InputStyled name='email' />
```
Comment (ibrahimkholil6210) [inline pages/auth/forgot-password.jsx:129]: @fazlerabby07 Please use the common component

**Review by ibrahimkholil6210 — CHANGES_REQUESTED** [review]
Please update the following comment

File: pages/auth/forgot-password.jsx (hunk 1 above)
Comment (fazlerabby07) [reply pages/auth/forgot-password.jsx:129]: okey

Comment (fazlerabby07) [reply pages/auth/forgot-password.jsx:63]: updated please check.

**Pull request merged**